pkg/queue/sqs.go

```diff
@@ -296,19 +315,13 @@ func (s *SQS) poll(key string, queueSpec QueueSpec) {
 				queueSpec.name, err)
 			return
 		} else {
-			klog.Fatalf("Unable to get approximate messages in queue %q, %v.",
+			klog.Errorf("Unable to get approximate messages in queue %q, %v.",
 				queueSpec.name, err)
 		}
 	}
 	klog.Infof("%s: approxMessages=%d", queueSpec.name, approxMessages)
 	s.queues.updateMessage(key, approxMessages)
 
-	if approxMessages != 0 {
-		s.queues.updateIdleWorkers(key, -1)
-		s.waitForShortPollInterval()
-		return
-	}
```

Comment:
Why was this change made? Please revert them. The changes to SQS code should not be part of this PR (if a change is to be made for them please raise a separate PR)